openhtf/io/proto/assembly.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
We're going to need it (a lot of why we want to use any of this is for genealogy) and I was going to add the functionality to openhtf. If you don't want it in openhtf, then I'll have to fork the `OutputToTestRun` code unfortunately